pkg/formats/cdx/cdx.go

```diff
@@ -0,0 +1,44 @@
+package cdx
```

Comment:
(see note about package names)
```suggestion
package cyclonedx
```

Reply:
will do

Reply:
Thank you :) :)